tests/topotests/pim_igmp_vrf/test_pim_vrf.py

```diff
@@ -323,7 +323,7 @@ def _test_vrf_pimreg_interfaces():
 
     r1 = tgen.gears["r1"]
     r1.vtysh_cmd("conf\ninterface blue\nip pim")
-    r1.vtysh_cmd("conf\nvrf blue\nip pim rp 192.168.0.11 239.100.0.1/32\nexit-vrf")
+    r1.vtysh_cmd("conf\nrouter pim vrf blue\nrp 192.168.0.11 239.100.0.1/32")
```

Comment:
While you are at it, can you please use the following more readable format
```
r1.vtysh_cmd(
"""
conf
router pim vrf blue
   rp 192.168.0.11 239.100.0.1/32
"""
)

```

Reply:
Will also merge the 2 vtysh invocations. When using multi-line, there's no point in calling vtysh_cmd() twice.